### DISCUSSION TITLE
How to add a button with a fontawesome icon

### DISCUSSION THREAD

**Discussion by mldgato** [rappasoft/laravel-livewire-tables#1795] [Q&A]
Hello, I have a table and I want to add a button that takes me to a route, I do this as follows:

`LinkColumn::make('Actions')
                ->title(fn ($row) => 'View')
                ->location(fn ($row) => route('admin.students.show', $row->id))
                ->attributes(fn ($row) => [
                    'class' => 'btn btn-primary btn-sm',
                ]),
`

But I need to not only see the "View" text but add a fontawesomen icon, for example:

View + Icon fas fa-info-circle

Thank you very much for reading and any guidance you can give me, greetings from Guatemala.

**Comment by mldgato** [comment]
I prefer not to place buttons anymore since I was asked that instead of placing a button, click anywhere in the row, it is not a solution but it works for me.

**Reply by lrljoe** [reply]
You can use a label type Column, and specify a view:

[https://rappasoft.com/docs/laravel-livewire-tables/v3/columns/available-methods#content-labels](https://rappasoft.com/docs/laravel-livewire-tables/v3/columns/available-methods#content-labels)

e.g.
```
Column::make('My one off column')
    ->label(
        fn($row, Column $column) => view('my.other.view')->withRow($row)
    ),
```

In that view, you'll have the full row, and you can style it as you wish!
